index.markdown

```diff
@@ -75,7 +75,7 @@ on how to appoach different concepts related to FAIR Digital Objects, i.e., Data
 target="metadatahub/index.html"
 background="assets/images/turntable.png"
 title="MetadataHub"
-subtitle="One to serve all"
+subtitle="A hub for repositories"
```

Comment:
A possible alternative might be "An access hub for repositories"

Reply:
+1 Access hub describes it even better, I think.